.github/workflows/sourcehold.yml

```diff
@@ -108,8 +138,13 @@ jobs:
     - name: Make artifacts
       shell: bash
       run: |
-        cd "$BUILD_ARTIFACT_PARENT_DIR_PATH"
-        zip -r ${{ github.workspace }}/${{ matrix.uploaded-artifact-name }}.zip "$BUILD_ARTIFACT_NAME"
+        if [[ ${{ matrix.os }} == 'windows-latest' ]]; then
+          cd "$BUILD_ARTIFACT_PARENT_DIR_PATH"
```

Comment:
We can place `cd "$BUILD_ARTIFACT_PARENT_DIR_PATH"` before `if` and remove it from `else`